content/bee-docs/installation/index.md

```diff
@@ -12,14 +12,15 @@ The client can be installed on MacOS and various Linux flavors.
 ## Installing the Bee binary
 At every official release, we release the binary files of our software. You can download them by navigating to [releases](https://github.com/ethersphere/bee/releases).
 
-A convenient way to download the binary is:
+A convenient way to download the binary is using `install.sh` script:
 
-```sh
-wget https://github.com/ethersphere/bee/releases/latest/download/bee-darwin-amd64 -O /usr/local/bin/bee
-chmod +x /usr/local/bin/bee
-```
+Grab the latest release:
```

Comment:
A comment:

The docs are made for the latest release (TAG). You are mentioning "latest release" here as `master`, whereas in fact you mean `edge`. 

Could you please correct that, and additionally also put emphasis on downloading the latest TAG (i.e. latest release). Suggesting to change the order (put downloading latest release (TAG) before instructions to download edge) and perhaps put a warning message for downloading the latest master (as the docs might not be up-to-date).

Reply:
@Eknir put a warning message where? when somebody executes install using script if its `edge` to show confirmation dialog? or just warn message?

Reply:
I don't think it is needed to put this warning message in the script itself. I meant more a message in the documentation, saying something along the lines that if the user installs the edge (master) version, then the documentation is not up-to-date with the latest changes. Additionally, you could mention that the network to which the nodes connects by default runs the latest release, so even the node might behave inconsistent.